classes/Sensors/System.php

```diff
@@ -266,4 +293,75 @@ public function WPUpdate($automatic)
             ));
         }
     }
+
+    /**
+     * Purge log files older than one month
+     */
+    public function LogFilesPruning()
+    {
+        if ($this->plugin->GetGlobalOption('purge-log', 0)) {
```

Comment:
@carmelopreeostudios let's make this more specific; something like wp-plugin-purge-404-log